### ISSUE TITLE
JSLint failing on spaces?

### ISSUE THREAD

**Issue by mythril** [douglascrockford/JSLint#68]
The following code fails to pass JSLint currently published on jslint.com (Edition 2011-09-15):

``` javascript
(function () {
    "use strict";

}());
```

With the error: 

> Problem at line 3 character 1: Unexpected ' '.
> Problem at line 3 character 1: Stopping. (75% scanned).

Now, most of the time, the prophet JSLint speaks the gospel truth, but I am a bit partial to spaces (assuming that's the real problem).
